src/subscript/fmu_copy_revision/fmu_copy_revision.py

```diff
@@ -0,0 +1,656 @@
+#!/usr/bin/env python
+#
+# fmu_copy_revision, for fast and secure copy for fmu revisions
+# using rsync as engine, with xargs to speed up multithreading
+#
+import sys
+from os.path import join
+import argparse
+import pathlib
+import tempfile
+import time
+import getpass
+import subprocess
+import shutil
+from multiprocessing import cpu_count
+
+
+import subscript
+from subscript import __version__
+
+logger = subscript.getLogger(__name__)
+
+DESCRIPTION = """This is a simple interactive script for copying a FMU revision folder
+with options of:
+    1. Selective copy, i.e. avoid data that can be regenerated
+    2. Speed up copying by multithreading
+    3. Retain correct file dates and user permissions
+
+Usage:
+    fmu_copy_revision  (for menu based input)
+
+     * or *
+
+    fmu_copy_revision --source 21.0.0 --target whatever --option 3 --threads 6 -cleanup
```

Comment:
Should we rename `option` here to `mode` or `copymode` or something?

Reply:
double dash in `cleanup`?

Reply:
> Should we rename `option` here to `mode` or `copymode` or something?

Or perhaps `profile`?